libenv/sysinfo.c

```diff
@@ -3482,6 +3486,12 @@ static void SysOSNameHuman(EvalContext *ctx)
                                       "Solaris", CF_DATA_TYPE_STRING,
                                       "source=agent,derived-from=solaris");
     }
+    else if (EvalContextClassGet(ctx, NULL, "amazon_linux") != NULL)
+    {
+        EvalContextVariablePutSpecial(ctx, SPECIAL_SCOPE_SYS, lval,
+                                      "Amazon Linux", CF_DATA_TYPE_STRING,
+                                      "source=agent,derived-from=amazon_linux");
```

Comment:
@nickanderson @larsewi should we also set the `AmazonLinux` class for backwards compatibility, in case someone is already using it?

Reply:
Yes, I guess we should

Reply:
Who would be using `AmazonLinux` already? Was that being defined by someting? In the MPF I only see `amazon_linux`.

Reply:
Oh I see in the comment now. Yeah, I guess we should make sure it's defined.